store/sqlstore/status_store.go

```diff
@@ -99,6 +102,89 @@ func (s SqlStatusStore) GetByIds(userIds []string) ([]*model.Status, error) {
 	return statuses, nil
 }
 
+// MySql doesn't has support for RETURNING clause or any euivalent so use transaction
+func (s SqlStatusStore) updateExpiredStatuses(t *gorp.Transaction) ([]*model.Status, error) {
+	selectQuery, selectParams, _ := s.getQueryBuilder().
+		Select("UserId, Status").
+		From("Status").
+		Where(
+			sq.Eq{"Status": model.STATUS_DND},
+			sq.Gt{"DNDEndTimeUnix": -1},
```

Comment:
Why here is `-1` and later is `1`?

Reply:
By the way, we usually use `0` for "unset timestamp", so... using `-1` here is not consistent with the rest of the application.